src/gui/qgsmapcanvas.cpp

```diff
@@ -202,6 +202,7 @@ QgsMapCanvas::QgsMapCanvas( QWidget * parent, const char *name )
   mCurrentLayer = NULL;
   mMapOverview = NULL;
   mMapTool = NULL;
+  mOldCursor = cursor();
```

Comment:
What is that for?
Normally the cursor state should be controlled by matching `QApplication::setOverrideCursor` and `QApplication::restoreOverrideCursor` calls.